src/Mutator/FunctionSignature/PublicVisibility.php

```diff
@@ -73,8 +73,7 @@ private function isBlacklistedFunction(string $name): bool
                 '__unset',
                 '__toString',
                 '__debugInfo',
-            ],
-            true
```

Comment:
not sure about this one. I guess it is a good practice to use the third strict param here. 

IMO it's better to have escaped mutant rather than not using strictness.

What do you think?

PS in the future, when we implement profiles (or something like that), we can instruct infection not to mutate true/false strict param in native functions